### ISSUE TITLE
I was trying to implement Slideshow. Faced some facts, might improve widget.

### ISSUE THREAD

**Issue by aaimaawebs** [jebster-pagekit/slideshow-widget#1]
Hello, I am Asharam Seervi from [Aaimaa Web Solutions](https://www.aaimaa.in/?utm_source=github&utm_medium=contribution).

We love Pagekit, Good to see your initiative for Slideshow. And, Yeah It's working fine but with some certain configuration only.

1. If checked " Transparent navbar as overlay" at pages. This slider will disappear. Console is generating few bugs. Screenshot of my console is attached.
![pagekit-slider-1](https://cloud.githubusercontent.com/assets/25567939/22864450/cd3d1638-f177-11e6-9ce9-c390570cabd5.PNG)

2. Even while selecting for Hero Section. It might appear in top section, I mean It looks like still in Top Section as well. Even checking "Full viewport height". 

Currently, Testing with Pagekit's theme One. Will contribute or let you know, if more bugs to repot.

**Comment by jeggy** [comment]
Thanks for the report, was a simple error by me. I just forget to add jquery as a dependency.

**Comment by aaimaawebs** [comment]
Thankyou so much @jeggy !!

This is our first bug reported. You made my day after accepting as bug and hence It's resolved now. Are you updated widget over Pagekit Marketplace?

**Comment by jeggy** [comment]
This is also my first bug fixed on github :P 
I updated it on the marketplace, just go to your extensions page and press update. 

But I realized that there aren't many themes that support full size width widgets, so it doesn't look so beautiful everywhere. I designed it for my [Bootstrap theme](https://github.com/jebster-pagekit/theme), which I haven't submitted to the marketplace which has support for fullsize width widgets.
